### PR TITLE
Amplitude

### LINKED ISSUE #51: App Analytics
Get Amplitude set up

https://github.com/ApollosProject/apollos-templates/blob/23ad15d32df3f122fc0192d6c5beb8e07c9b94f4/docs/AMPLITUDE.md

reference implementation: https://github.com/watermarkchurch/theporch-apollos/blob/fd2653cedaee1c70a2852396d768382e3dc773ae/ThePorch/src/Providers.js#L11

### PR DESCRIPTION
**Pull request by gburgett**
* Hooks up amplitude
* Sends click/view events for everything I could think of

fixes #51

### CODE REVIEW

**Comment by gburgett** [comment]
@jpowell it looks like we already have the AMPLITUDE_API_KEY set up in secrets? Is that correct?
I don't think I need to do anything else to configure it once we merge

File: apolloschurchapp/src/Providers.tsx [hunk 1]
```diff
@@ -30,7 +36,19 @@ const AppProviders: React.FunctionComponent<any> = (props) => {
             })
           }
         >
-          <AnalyticsProvider>
+          <AnalyticsProvider
+            // Not using server-side analytics (segment, GA, rock), using Amplitude instead
+            useServerAnalytics={false}
+            trackFunctions={[
```
Comment (jpowell) [inline apolloschurchapp/src/Providers.tsx:42]: does this not work for all things? dumb question here...

**Review by jpowell — APPROVED** [review]
good job - makes sense that you'd have to use the hook to actually track events... 🤦🏽‍♂️

**Comment by gburgett** [comment]
And now we wait 30 minutes before we can test it on my phone...

**Pull request merged**